app/incident/reports/controller.js

```diff
@@ -145,6 +147,12 @@ export default AbstractReportController.extend(UserSession, NumberFormat, {
 
   actions: {
     generateReport() {
+
+      if (isEmpty(get(this, 'endDate'))) {
+        let now = new Date();
```

Comment:
Should this me a moment object or raw Date?

cc: @MatthewDorner whats your thought?

Reply:
I believe you'd have to turn it into a Date anyway as that's what the Ember date attribute uses, so it seems ok. I'm more wondering if we need to think about time zones.

Reply:
Note that we also changed the behavior of the DB query here, as it used to leave the end date open meaning it would get everything but now we set the end date to the end of current day as it is in user's local time (I think).